src/frontend/komponenter/Behandling/Vurdering/VisEllerEndreVurdering.tsx

```diff
@@ -60,7 +59,11 @@ const VisEllerEndreVurdering: FC<Props> = ({
             );
         case Redigeringsmodus.VISNING:
             return (
-                <VisVurdering vurdering={vurdering} settRedigeringsmodus={settRedigeringsmodus} />
+                <VisVurdering
+                    vurdering={vurdering}
+                    settRedigeringsmodus={settRedigeringsmodus}
+                    lagreVurdering={lagreVurdering}
```

Comment:
Det føles litt rart att komponenten  `VisVurdering` skal få med en metode som heter `lagreVurdering` når den egentlige kun skal vise data. 
Burde man kalle det argumentet "nullstillVurdering" ?

Reply:
Fikset! Endrer til "resetVurdering" for at metodenavnet ikke skal kræsje med en annen importert metode.